megaraid.py

```diff
@@ -32,11 +38,11 @@ def check_adapter(self, instance, megacli):
                 current_adapter = line.split(' ')[1]
                 adapters['0'] = dict()
             if line.startswith('State'):
-                if "Optimal" in line:
+                if 'Optimal' in line:
                     adapters[current_adapter]['state'] = 1
                 else:
-                    adapters[current_adapter]['state'] = 0
-                    self.send_megaraid_alert('Adapter %s state is not optimal: %s' % (current_adapter, adapters[current_adapter]['state']), current_adapter)
+                    adapters[curent_adapter]['state'] = 0
```

Comment:
I think there is  a typo here

Reply:
Did you mean double quotes in "Optimal" ?

Reply:
No. `curent` instead of `current`.

Reply:
ah, sorry, didn't notice that